source/databricks/calculation_engine/package/calculation_input/table_reader.py

```diff
@@ -49,13 +50,22 @@ def __init__(
         else:
             self._time_series_points_table_name = time_series_points_table_name
 
+        if metering_point_periods_table_name is None:
+            self._metering_point_periods_table_name = (
+                paths.METERING_POINT_PERIODS_TABLE_NAME
+            )
+        else:
+            self._metering_point_periods_table_name = metering_point_periods_table_name
```

Comment:
Not sure whether this is better...
```suggestion
        self._metering_point_periods_table_name = metering_point_periods_table_name or (
                paths.METERING_POINT_PERIODS_TABLE_NAME
            )
```

Reply:
if we do this, should i not do the same for time series period table name

Reply:
That would be nice :-)

Reply:
i change both to "or" syntax